generate.py

```diff
@@ -46,7 +54,7 @@
         }
 
         # Append to skeleton
-        meta["modules"].append(module)
+        meta.get("modules").append(module)
```

Comment:
Can you leave this as it was?

Reply:
Type an non existing object an see what happens

https://www.w3schools.com/python/trypython.asp?filename=demo_ref_dictionary_get

Reply:
No matter whether you use `.get` or a simple index-like syntax (`["modules"]`), an exception gets thrown if the `modules` key does not exist. Why switch to `get` in such a case?